syntax = "proto3";

package Seraphim.Face.FaceDetector;

import "Types.proto";

/*
 * Requests
 *
 * Client --> Server
 */

message DetectionRequest {
  Types.Image2D image = 1;
  Types.Region2D roi = 2;
}

/*
 * Responses
 *
 * Server --> Client
 */

message DetectionResponse {
  repeated Types.Region2D faces = 1;
}
